Tasks/ANTPreview/ant.ps1

```diff
@@ -116,8 +109,26 @@ else
 # check if code coverage has been enabled
 if($isCoverageEnabled)
 {
-   Write-Verbose "Calling Publish-CodeCoverage" -Verbose
-   Publish-CodeCoverage -CodeCoverageTool $codeCoverageTool -SummaryFileLocation $summaryFile -ReportDirectory $reportDirectory -Context $distributedTaskContext    
+   # run report code coverage task which generates code coverage reports.
+   Write-Verbose "Collecting code coverage reports" -Verbose
+   try
+   {
+		Invoke-Ant -AntBuildFile $antBuildFile -Targets $CCReportTask 
+   }
+   catch
+   {
+		Write-Warning "Failed to collect code coverage. There might be no tests." -Verbose
```

Comment:
If you mean waarning. Yes i can see the warning when the report collection fails